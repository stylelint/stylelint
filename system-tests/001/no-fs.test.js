'use strict';

const stylelint = require('../../lib');
const { caseConfig, caseCode, prepForSnapshot } = require('../systemTestUtils');

const CASE_NUMBER = '001';

it('no-fs - valid sanitize.css and their config', async () => {
	expect(
		prepForSnapshot(
			await stylelint.lint({
				code: await caseCode(CASE_NUMBER),
				config: await caseConfig(CASE_NUMBER),
			}),
		),
	).toMatchSnapshot();
}, 10000);
