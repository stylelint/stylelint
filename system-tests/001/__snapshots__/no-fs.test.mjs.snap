// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no-fs - valid sanitize.css and their config 1`] = `
{
  "cwd": "",
  "errored": false,
  "output": [
    {
      "deprecations": [],
      "errored": false,
      "invalidOptionWarnings": [],
      "parseErrors": [],
      "source": "/path/to/dummy.css",
      "warnings": [],
    },
  ],
  "report": [
    {
      "deprecations": [],
      "errored": false,
      "invalidOptionWarnings": [],
      "parseErrors": [],
      "source": "/path/to/dummy.css",
      "warnings": [],
    },
  ],
  "reportedDisables": [],
  "results": [
    {
      "deprecations": [],
      "errored": false,
      "ignored": undefined,
      "invalidOptionWarnings": [],
      "parseErrors": [],
      "source": "/path/to/dummy.css",
      "warnings": [],
    },
  ],
  "ruleMetadata": {
    "at-rule-empty-line-before": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/at-rule-empty-line-before",
    },
    "at-rule-no-unknown": {
      "url": "https://stylelint.io/user-guide/rules/at-rule-no-unknown",
    },
    "block-no-empty": {
      "url": "https://stylelint.io/user-guide/rules/block-no-empty",
    },
    "color-hex-length": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/color-hex-length",
    },
    "color-no-invalid-hex": {
      "url": "https://stylelint.io/user-guide/rules/color-no-invalid-hex",
    },
    "comment-empty-line-before": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/comment-empty-line-before",
    },
    "comment-no-empty": {
      "url": "https://stylelint.io/user-guide/rules/comment-no-empty",
    },
    "comment-whitespace-inside": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/comment-whitespace-inside",
    },
    "custom-property-empty-line-before": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/custom-property-empty-line-before",
    },
    "declaration-block-no-duplicate-properties": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/declaration-block-no-duplicate-properties",
    },
    "declaration-block-no-shorthand-property-overrides": {
      "url": "https://stylelint.io/user-guide/rules/declaration-block-no-shorthand-property-overrides",
    },
    "declaration-block-single-line-max-declarations": {
      "url": "https://stylelint.io/user-guide/rules/declaration-block-single-line-max-declarations",
    },
    "declaration-empty-line-before": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/declaration-empty-line-before",
    },
    "font-family-no-duplicate-names": {
      "url": "https://stylelint.io/user-guide/rules/font-family-no-duplicate-names",
    },
    "font-family-no-missing-generic-family-keyword": {
      "url": "https://stylelint.io/user-guide/rules/font-family-no-missing-generic-family-keyword",
    },
    "function-calc-no-unspaced-operator": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/function-calc-no-unspaced-operator",
    },
    "function-linear-gradient-no-nonstandard-direction": {
      "url": "https://stylelint.io/user-guide/rules/function-linear-gradient-no-nonstandard-direction",
    },
    "function-name-case": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/function-name-case",
    },
    "keyframe-declaration-no-important": {
      "url": "https://stylelint.io/user-guide/rules/keyframe-declaration-no-important",
    },
    "length-zero-no-unit": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/length-zero-no-unit",
    },
    "media-feature-name-no-unknown": {
      "url": "https://stylelint.io/user-guide/rules/media-feature-name-no-unknown",
    },
    "no-duplicate-at-import-rules": {
      "url": "https://stylelint.io/user-guide/rules/no-duplicate-at-import-rules",
    },
    "no-duplicate-selectors": {
      "url": "https://stylelint.io/user-guide/rules/no-duplicate-selectors",
    },
    "no-empty-source": {
      "url": "https://stylelint.io/user-guide/rules/no-empty-source",
    },
    "no-invalid-double-slash-comments": {
      "url": "https://stylelint.io/user-guide/rules/no-invalid-double-slash-comments",
    },
    "property-no-unknown": {
      "url": "https://stylelint.io/user-guide/rules/property-no-unknown",
    },
    "rule-empty-line-before": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/rule-empty-line-before",
    },
    "selector-pseudo-class-no-unknown": {
      "url": "https://stylelint.io/user-guide/rules/selector-pseudo-class-no-unknown",
    },
    "selector-pseudo-element-colon-notation": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/selector-pseudo-element-colon-notation",
    },
    "selector-pseudo-element-no-unknown": {
      "url": "https://stylelint.io/user-guide/rules/selector-pseudo-element-no-unknown",
    },
    "selector-type-case": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/selector-type-case",
    },
    "selector-type-no-unknown": {
      "url": "https://stylelint.io/user-guide/rules/selector-type-no-unknown",
    },
    "string-no-newline": {
      "url": "https://stylelint.io/user-guide/rules/string-no-newline",
    },
    "unit-no-unknown": {
      "url": "https://stylelint.io/user-guide/rules/unit-no-unknown",
    },
    "value-keyword-case": {
      "fixable": true,
      "url": "https://stylelint.io/user-guide/rules/value-keyword-case",
    },
  },
}
`;
